presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -73,6 +73,8 @@
     public static final String ENABLE_INTERMEDIATE_AGGREGATIONS = "enable_intermediate_aggregations";
     public static final String PUSH_AGGREGATION_THROUGH_JOIN = "push_aggregation_through_join";
     public static final String PUSH_PARTIAL_AGGREGATION_THROUGH_JOIN = "push_partial_aggregation_through_join";
+    public static final String FILTER_AND_PROJECT_MIN_OUTPUT_PAGE_SIZE = "filter_and_project_min_output_page_size";
```

Comment:
Those properties are nice to have if you would like to disable this feature at some point. We can remove them once we make sure that the feature doesn't break anything.